sdk/src/androidTest/java/ly/count/android/sdk/ModuleUserProfileTests.java

```diff
@@ -588,6 +588,72 @@ public void internalLimit_setProperties_maxValueSize() throws JSONException {
ow2Pd5maVoZLwsqn28XwmsPmPCyxAXeV3nQZYkl2kOaVQLJDAeJsRmRXPUKJA9XA7oS9ddEDzbum5l4z57JVaXZjl90JT4O8npWDIYF2Mvnf1kmCwoQEnvM0KkrMRcivUnPxcYYxD1fbvl80rQI1XYuLul9GTHUJ4nps5qFVwcKeJQEsOlesvtprpgaKlQkXLgZ18wkc5uyxk8dDH5BcCyYN6Oi4EXe5GFMwMh51FLHI8loIYUR27sAPhPyEkmqbAipgXkSItlUcZ5F8UJgZO2Ud12qrVRYwvOqBHS07ys2nOZJyQMLQxPf5PxNyE8sehBG85PWjUB3Phd3jPpUQfU5CnXywZu90lzICZadGssvewqvcp1QTOKRmGGrpr9emwYDLr1t3E3Pvf5Tu8AVGAY4racg3mU0Rmc43kg9a4fJJENl0OdipewaVPg9yeo3WEWVZlPDM9nfben37QEzLfxwRE7TDtNEuMfgjMDzk74FGzi50X5GXLwKnfUnp4WCwphTJo6QgtXCIp8uNCX1gAKKFNPPT8425CLIPcNYySYGSv0QUM9GfSObZsIt44IM88VRRVXoGjpuxcAyrPDUDqmIu6AACBecZhKaJhaZkchoxPJgDtIpxkW2qBVyEC6PmdvVz29qPJloqnKNkzaG3wXzlSKu6jttln1oNQ5t5YLaUjv0fuxlI2hv4jbflbvqTBshGRyIbQq8NCAcXbGZEINeXNg8ezzW9almBhaEFVyEElEPhnas8cb9QncVWUFhha9zcx5QrNMP8FOJiGxiBBGK2g5S74ZwL6LrqXYTrJ456tmlYe9xlSgooZaUNpr6ZQ3MhcR4Ozo3HtoNujA5f5ZeN1Mkx8lIrnlDjsQsaioGLIAtwlTmHC1j42eLcT4mvIaaTRfy4xhxMH3Frb9UQQP4gm2LitatYcYK9PIHV6c4kvpgoYMMg8f58ovq21AXwhxOmVSsVrYXAslPKTZhe6C2ZnYCZaykzI1CpUIrtG5y0PvxsDDIFFtXOml74KchEq2DSBOB6FNJCroExNFkU2L7aD03FrR9qRgwyO1aXDuug1yDbzoUTZpyulmScxcqwKRn2GA26YuPuQYcMQIUdkgd5lCgeNhsQwjByB2jzLXJmDPyTywhLIWZolpVskZ6s32hH2qvAHrmsAXtBj6URiiBWL2HkPnY4oZpZDLe8I2ZkX6jqUQvlj7dsf72xRTxVKFkR7v3gbGP1hHyEoFfYOD5yTLmW1J3WrxI28cRTRc52UPRtzpYudFawodE6TopPkUUFE565b0om1E9YfcZ3I2qPGjvID2fuQjKtlwXesL0uDNFXK7dPpndiV8A0iaJNs83ZIDD3L4c9WTM1DDiqdU4WeatnnUlg5rTApzbw5Ro51Lm3hi6NWWXVifC9S3kjNax6OXfbS8Rwv2rVJvfPCrzbqddExkgv9B6Vw4q69eF3cKr4bIB2Wci3BhvqyRsqiHEP6BdvMIfKAVdzqUi5o7aBq0jR6pyiUQtzzKmXdanMLFlDZNEnAAjlOkQG0fhWbN4PtpGMt5U5xtWUtyVMuMjDQncmyt3BuRqDJ2cRNeF590IBoKEm0jEw4ihBlXWbJfb9uuvUqenIlIVJg7CJdrQFsjPP727IgJqPfhayIzJYjpl9sPmzhTKar4A0M3iIUjiaeZXXLwnUIMCyDiG3Lc5OG4awPoui6KBrcfW03JUPlI4HdpOigJsTXTi39fufdccwsFuWdd7UzKJIPD5RFH8Y64CIDjfuY7byZaSj7XJmiDRL9sevjXulkRivCtSRV3sgsdrBCJbdfltWNJLUwNQzIAXuejXZhLkYNud5rLHWO9ZPcCfFAoTKx8qwlCF5r1v2cdNWjcBsh1aWOVcIMC8NL2zJqYxeYVFtSBuTYjD4k68cVTKjgfQXuawEk5S0EHLj1SwY3jHODXdZxiyd2PARTBZsekdG8GE54JWuPtzUGBiWW2pPRV7wiTHWglp9xbHfQEA7U3mDxRz4mb0VLiitgy0wu7wWxksT8JMyZICP3eK2Ncsa80Olttxa1p8EoAckncyRssxw03swlYtJQ3ccABDhXm76QtApDpgcnepOVIiHeIFK1g3f6Z6yojEqVoKoJjvu6InsXszWcmvbFDqOH4NIJdd3yULyJ5UPApVMJhfWXWp1SRB12bzs7YrO26QBFyiWeg0d17WTrdUelBx7KWWrxPibNBJFZyrZLyAwa53VnLuALIyWdP6LlOcIeCGkywXML16FiaPFby22T5D6QyvVvFfj5lCBWOY3T23XBDEMFyhi5pBsaUAN1ur9EQjk4zCx4dOOaelFF3rVZH69qAr03kxnrRQFWhPQVcw64NpjfVKhWZkRdddg4VIz2MOZGMPCc9loCaPmHhOlPFBqaFPKuDJnJCw38EssEc55jTzinBllGXpswjWuAKdQqUv5Aiq885OmtwsIXHT0hmGi80pmSscYdmKiYntsL31BVpPAtCqiPcnnGoXRwbV2v8KwVLhcBzzeNQ8MZRq5DZRSzX3H6fZ7kypuEPkxuPEhOPREv3fVNQTmSMjPJEw5EmSC3E9cZlQzGCzy9EAJcU2jsEWwPEszwYwL7q2ylbvpvOeguvge5IpCUsLq9DIrI43W7FSXTMUCRCCJx5V17qxya7kby5P6S6OBzV3xSvK0mFtIs1Y6el0MGRUbuSpQ6fYheNLjzlbyBOdCXGDNt8DlzrHwkmoi9R6kirwIu6YtkWTMmuXoRaZh4EAmxcjdhoX2PfnpJosLe8hQaJZHaQxwUB7j2nSVZBNj2DrTDrZVvVvccChxDLUJ0ngkIvMJvTdriDZYR6tSyMvBRFUCzKzBelxH0DoCBRQCg64JKe5dEdRaQ074M4HNixZAOOlsl8KYQaReBkeGPFIQv9ekpzM1mwjnSEM11tOFokEltflGax5Y7WRKaZvZClMGLSApaNakvtX6sQHCTv7aCikwTvthRXgGXQPHu7YvKEKBoinOaNKZ08htxaI"
```

Comment:
In a way this feels weird to look at. Is randomly generating a string with the desired length and also validating the string within the test is a bad idea?

Reply:
Actuallu it is really good to have. It would probably benefit for other tests also. Thx on it